cmd/agent/app/reporter/tchannel/flags.go

```diff
@@ -30,6 +30,8 @@ const (
 	hostPort                  = "host-port"
```

Comment:
> `collectorHostPort         = "collector.host-port"`

I think this was renamed to remove "collector" part. Are you missing a re-base?

Reply:
`collector` was removed only from the new flags - `reporter.grpc.collector.host-port`

Reply:
ok. This still needs a rebase, lots of merge issues.